src/generic/grideditors.cpp

```diff
@@ -1266,14 +1245,26 @@ void wxGridCellBoolEditor::SetSize(const wxRect& r)
 
     // position it in the centre of the rectangle (TODO: support alignment?)
 
+    wxRect cellRect = r;
+
 #if defined(__WXGTK__) || defined (__WXMOTIF__)
+#if defined(__WXGTK3__)
+    cellRect.x += 6;
+    cellRect.width -= 12;
```

Comment:
This should be written as `cellRect.Deflate(6, 0)` but, more importantly, where does this `6` come from? Is it the same one as in `wxSizerFlags::GetDefaultBorder()`?